sensors/2.0/iiohal_mediation_v2.0/libiio_client/iiod-client.c

```diff
@@ -605,25 +658,31 @@ ssize_t iiod_client_read_unlocked(struct iiod_client *client, void *desc,
     iio_snprintf(buf, sizeof(buf), "READBUF %s %lu\r\n",
             iio_device_get_id(dev), (unsigned long) len);
 
-    ret = iiod_client_write_all(client, desc, buf, strlen(buf));
-    if (ret < 0)
-        return ret;
+	ret = iiod_client_write_all(client, desc, buf, strlen(buf));
+	if (ret < 0) {
+		ERROR("WRITE ALL: %zu\n", ret);
+		return ret;
+	}
 
     do {
         int to_read;
 
-        ret = iiod_client_read_integer(client, desc, &to_read);
-        if (ret < 0)
-            return ret;
-        if (to_read < 0)
-            return (ssize_t) to_read;
-        if (!to_read)
-            break;
-
-        if (mask) {
-            ret = iiod_client_read_mask(client, desc, mask, words);
-            if (ret < 0)
-                return ret;
+		ret = iiod_client_read_integer(client, desc, &to_read);
+		if (ret < 0) {
+			ERROR("READ INTEGER: %zu\n", ret);
+			return ret;
+		}
+		if (to_read < 0)
+			return (ssize_t) to_read;
+		if (!to_read)
+			break;
+
+		if (mask) {
+			ret = iiod_client_read_mask(client, desc, mask, words);
+			if (ret < 0) {
+				ERROR("READ ALL: %zu\n", ret);
```

Comment:
Seems we are adding only error log here. If it is only for debug purpose, this code log can be removed